src/components/v2/appDetails/k8Resource/nodeDetail/nodeDetail.api.ts

```diff
@@ -13,20 +16,7 @@ export const getManifestResource = (
     isResourceBrowserView?: boolean,
     selectedResource?: SelectedResourceType,
 ) => {
-    if (
-        ad.appType === AppType.EXTERNAL_HELM_CHART ||
-        ad.deploymentAppType === DeploymentAppType.helm ||
-        isResourceBrowserView
-    ) {
-        return getManifestResourceHelmApps(ad, podName, nodeType, isResourceBrowserView, selectedResource)
-    }
-    const cn = ad.resourceTree.nodes.filter((node) => node.name === podName && node.kind.toLowerCase() === nodeType)[0]
-
-    return get(
-        `api/v1/applications/${ad.appName}-${ad.environmentName}/resource?version=${cn.version}&namespace=${
-            cn.namespace || ''
-        }&group=${cn.group || ''}&kind=${cn.kind}&resourceName=${cn.name}`,
-    )
+    return getManifestResourceHelmApps(ad, podName, nodeType, isResourceBrowserView, selectedResource)
```

Comment:
can we directly use it on the places instead of returning from this method?

Reply:
replaced and removed the the unnecessary function